### DISCUSSION TITLE
Ideas PLA clogs?

### DISCUSSION THREAD

**Discussion by ManuelPeiro** [supermerill/SuperSlicer#3023] [General]
Hi,

I know how difficult this question is, but maybe someone knows, or can give me ideas...

The problem is that when printing with PLA, with ABS and PETG no, I have jams... I use a Ratrig v3 printer, which has profiles including Super Slicer, without modifying, only infill and shells, and the retractions that I have changed to firmware .

What is also curious is that with another slicer, Cura, with the most similar possible options to Super slicer, the piece is printed.

I am trying it with a small piece approximately 5x5x5 cm, because with SuperSlicer, in the 3rd or 4th layer the extruder gets clog...

The temperatures I use are 195/70 for PLA, but I have tried several, and the same problem.
The extruder is an Orbiter and the hotend is a Rapido (short version).

If someone can give me ideas... what makes Super Slicer different from Cura? that with one there are clogs (SS) and with Cura no, at first I thought about the retractions, and that's why I changed them to firmware... but a bit the same... except that Super Slicer, makes more retractions than Cura...

Any ideas?

**Comment by supermerill** [comment]
probably a very low flw somewhere.
look for ironing, and very thin gapfill.

**Reply by ManuelPeiro** [reply]
Well.... I'll try, it's very possible. I just looked at it and the presets I'm using, which come in Super Slicer for the printer, generally use low extrusion multipliers... being especially low for PLA.

Defaults extrusions multiplier are:

ABS - 0.98
PETG HF - 0.94
PETG v6 - 0.94
PLA --> 0.89 <--
TPU -0.89
